build_library/grub.cfg

```diff
@@ -19,7 +19,7 @@ set randomize_disk_guid=""
 set oem_id=""
 
 # Anything else the OEM adds should use this variable.
-set linux_append="systemd.unified_cgroup_hierarchy=false systemd.legacy_systemd_cgroup_controller=false"
+set linux_append="systemd.unified_cgroup_hierarchy=true"
```

Comment:
Instead of changing this here, can we remove this parameter and only change the flag in the systemd ebuild file?

Reply:
fair